pulpcore/cli/ostree/distribution.py

```diff
@@ -74,11 +74,13 @@ def distribution(ctx: click.Context, pulp_ctx: PulpCLIContext, distribution_type
 @pass_pulp_context
 def update(
     pulp_ctx: PulpCLIContext,
-    distribution_ctx: PulpOstreeDistributionContext,
+    distribution_ctx: PulpEntityContext,
     base_path: Optional[str],
     repository: Optional[Union[str, PulpEntityContext]],
     version: Optional[int],
 ) -> None:
+    assert isinstance(distribution_ctx, PulpOstreeDistributionContext)
```

Comment:
Was this necessary? I am not a fan of writing asserts because of type change.

Reply:
Yes, this is necessary to hint that the passed context object is really of that type.

Reply:
I meant the whole change, with PulpEntityContext.

Reply:
I know. but the typing hints of click improved and so that is needed too.